content/opt/setup/11-timezone

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+. /opt/helpers
+
+[ -n "${TZ-}" ] || exit 0
+
+echo "=> Setting timezone to ${TZ}"
+
+TZFILE="/usr/share/zoneinfo/${TZ}"
+if [ ! -f ${TZFILE} ]; then
+	echo "Timezone ${TZ} not available, using default timezone"
+	exit 0
+fi
+
+# As of Debian Jessie it's not enough to update /etc/timezone; the
```

Comment:
I never heard of using `/etc/timezone`.  Why is this necessary?

Reply:
The only usable info I found is this thread on stackexchange, which says that java may use it: https://unix.stackexchange.com/a/452566

libc just requires `/etc/localtime`.

Reply:
Tested in a container, this suffices:

```
 ln -sf "${TZFILE}" /etc/localtime
 dpkg-reconfigure -f noninteractive tzdata
```

`tzdata` reconfiguration listens to `/etc/localtime` setting and updates `/etc/timezone` accordingly. No need to put code in this script and waste brainpower in a few years to ask oneself: Why is this here?

Reply:
Not sure why I thought it was necessary to update /etc/timezone. tzdata's reconfigure script does it for you as long as you set the symlink.